libraries/GCS_MAVLink/GCS.h

```diff
@@ -754,6 +945,11 @@ class GCS
                               ap_var_type param_type,
                               float param_value);
 
+    static MissionItemProtocol_Waypoints *_missionitemprotocol_waypoints;
```

Comment:
Why do these need to be deferred as pointers?

Reply:
Because I don't want to have to check within them that they have a valid mission pointer to work with; these are only instantiated if AP_Mission* is non-null.